src/Tasks/Common/Resources/Strings.resx

```diff
@@ -698,4 +698,52 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1148: Found multiple publish output files with the same relative path: {0}.</value>
     <comment>{StrBegin="NETSDK1148: "}</comment>
   </data>
+  <data name="CrossgenToolMissingInPDBCompilationMode" xml:space="preserve">
+    <value>NETSDK1149: CrossgenTool not specified in PDB compilation mode</value>
```

Comment:
It seems the convention here is to end each error message with a period.

Reply:
Fixed in 6th commit, thanks for pointing that out.